lib/core/class-groups-group.php

```diff
@@ -67,6 +67,17 @@ public function __get( $name ) {
 		global $wpdb;
 		$result = null;
 		if ( $this->group !== null ) {
+
+			/**
+			 * Filter the group properties.
+			 * @param WP_User
+			 * @param string
+			 */
+			$null = apply_filters( 'groups_group_get_' . $name, null, $this->group, $name );
```

Comment:
reversed approach - should be applied on result instead

Reply:
This method is essentially a shortcut which will prevent any unnecessary logic to run and take up resources.
The same method is used in WP core many time.

Reply:
I would apply similar considerations related to short-circuit filters as I mentioned in https://github.com/itthinx/groups/pull/59/files#r1450356957 but particularly on this instance there is a lack of context for the filter - in what context is the property accessed - which can lead to erroneous uses of the filter that can have undesired side-effects on the normal processing.

I wonder whether this would still be needed for https://github.com/JoryHogeveen/view-admin-as/blob/master/modules/class-groups.php or if the addition of the (modified) groups_user_is_member filter is sufficient to cover the needs of that module.